app/client/src/components/InstallChaos/index.tsx

```diff
@@ -42,27 +69,94 @@ export function InstallChaos(props: InstallProps) {
 		setTimeout(() => setCopying(false), 3000);
 	}
 
+	function showYamlInPage(text: string) {
+		if (viewing === false) {
```

Comment:
instead of this check can we use (!viewing)

Reply:
Okay i changing it.